src/migrations.js

```diff
@@ -0,0 +1,31 @@
+import { produce, setAutoFreeze } from 'immer';
+
+export const migrate = (
+  data,
+  migrations,
+) => {
+  setAutoFreeze(false);
+
+  let version = data._migrationVersion || -1;
```

Comment:
Grab the last migration that has run, and if we've never run a migration before set an initial value